From 328a88a4fe0c70fb17d1628647435428609d8361 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sun, 13 Mar 2022 18:05:24 +0800
Subject: [PATCH 1/2] driver: pvr_buffer_sync: replace dma_resv_get_list with
 dma_resv_shared_list

The kernel commit fb5ce730f21434d8(dma-buf: rename and cleanup dma_resv_get_list v2)
renamed the dma_resv_get_list to dma_resv_shared_list.
The kernel commit 6edbd6abb783d54(dma-buf: rename and cleanup dma_resv_get_excl v3)
used dma_resv_excl_fence as function name

Upstream-Status: Inappropriate [upstream is still using k5.10 version]

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 services/server/env/linux/pvr_buffer_sync.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/services/server/env/linux/pvr_buffer_sync.c b/services/server/env/linux/pvr_buffer_sync.c
index c5f0344..c9c2de5 100644
--- a/services/server/env/linux/pvr_buffer_sync.c
+++ b/services/server/env/linux/pvr_buffer_sync.c
@@ -192,8 +192,8 @@ pvr_buffer_sync_pmrs_fence_count(u32 nr_pmrs, struct _PMR_ **pmrs,
 		if (WARN_ON_ONCE(!resv))
 			continue;
 
-		resv_list = dma_resv_get_list(resv);
-		fence = dma_resv_get_excl(resv);
+		resv_list = dma_resv_shared_list(resv);
+		fence = dma_resv_excl_fence(resv);
 
 		if (fence &&
 		    (!exclusive || !resv_list || !resv_list->shared_count))
@@ -251,8 +251,8 @@ pvr_buffer_sync_check_fences_create(struct pvr_fence_context *fence_ctx,
 				goto err_destroy_fences;
 		}
 
-		resv_list = dma_resv_get_list(resv);
-		fence = dma_resv_get_excl(resv);
+		resv_list = dma_resv_shared_list(resv);
+		fence = dma_resv_excl_fence(resv);
 
 		if (fence &&
 		    (!exclusive || !resv_list || !resv_list->shared_count)) {
-- 
2.36.1

